src/main/java/com/faforever/api/deployment/ExeUploaderController.java

```diff
@@ -0,0 +1,59 @@
+package com.faforever.api.deployment;
+
+import com.faforever.api.config.FafApiProperties;
+import com.faforever.api.error.ApiException;
+import com.faforever.api.error.Error;
+import com.faforever.api.error.ErrorCode;
+import com.google.common.io.Files;
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiResponse;
+import io.swagger.annotations.ApiResponses;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.multipart.MultipartFile;
+
+import static org.springframework.http.MediaType.APPLICATION_JSON_UTF8_VALUE;
+
+@RestController
+@RequestMapping(path = "/exe")
+@Slf4j
+public class ExeUploaderController {
+  private final FafApiProperties apiProperties;
+  private final ExeUploaderService exeUploaderService;
+
+  public ExeUploaderController(
+    FafApiProperties apiProperties,
+    ExeUploaderService exeUploaderService
+  ) {
+    this.apiProperties = apiProperties;
+    this.exeUploaderService = exeUploaderService;
+  }
+
+  @ApiOperation("Upload an exe file")
```

Comment:
"and override the existing one for the current version"

Reply:
Can you explain, please? I didn't get what you mean.